jwst/lib/set_telescope_pointing.py

```diff
@@ -204,6 +210,37 @@ def add_wcs(filename, default_pa_v3=0., siaf_path=None, engdb_url=None,
     model.close()
     logger.info('...update completed')
 
+def update_mt_kwds(model):
+    """Add/update the Moving target header keywords
+
+    If the target type is "moving_target" check for the moving target position
+    table. If this is available calculate the moving target position keywords
+    and insert or update MT_RA & MT_DEC.
+    """
+
+    if model.extra_fits.hasattr('MOVING_TARGET_POSITION'):
+        time_mt = model.extra_fits.MOVING_TARGET_POSITION.data.time
+        exp_midpt_mjd = model.meta.exposure.mid_time #- 0.207
+        # check to see if the midpoint of the observation is contained within
+        # the timerange of the MT table
+        if time_mt[0] <= exp_midpt_mjd <= time_mt[-1]:
+            ra = model.extra_fits.MOVING_TARGET_POSITION.data.moving_target_RA
+            dec = model.extra_fits.MOVING_TARGET_POSITION.data.moving_target_Dec
+            f_ra = interp1d(time_mt,ra)
```

Comment:
Huh, surprised flake8 didn't complain about the lack of whitespace after the comma in the argument list ...

Reply:
Our `flake8` checks are lenient here because of all our `numpy` code that does things like `array[0,1,2]` instead of `array[0, 1, 2]`.  🤷‍♀️